### DISCUSSION TITLE
use edge attribute in HGT

### DISCUSSION THREAD

**Discussion by Everyday-seu** [pyg-team/pytorch_geometric#4366] [General]
Thanks for your fantastic jobs about pytorch_geometric!!!
Notice that existing HGTConv method uses x_dict,edge_index_dict, but I want to use edge attribute as well.
What should I do?
Look forward to your reply!

**Comment by rusty1s** [comment]
Please see my answer here: https://github.com/pyg-team/pytorch_geometric/discussions/4364
